CHANGELOG.md

```diff
@@ -5,6 +5,9 @@ The format is based on [Keep a Changelog](http://keepachangelog.com/en/1.0.0/)
 and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.html).
 
 ## [1.2.0] - 2024-07-02
+### Added
+- [astarte_appengine_api] Implement property-based tests for `cast_value` functions.
+
```

Comment:
This release is already released. This will have to be added in the next unreleased version, 1.2.1.
Usually start developing on a new release with a commit that bumps all versions in mix files and updates the README. Have a look at the git history for such commits